pytorch_to_returnn/torch/nn/modules/normalization.py

```diff
@@ -53,6 +53,45 @@ def import_params_torch_to_returnn(self, *, layer: LayerNormLayer, torch_module:
     layer.params["bias"].load(torch_module.bias.detach().numpy(), session=session)
 
 
+class GroupNorm(Module):
+  def __init__(self, num_groups: int, num_channels: int, eps: float = 1e-5, affine: bool = True) -> None:
+    super(GroupNorm, self).__init__()
+    assert num_groups == 1 or num_groups == num_channels, "Not implemented otherwise"
+    self.num_groups = num_groups
+    self.num_channels = num_channels
+    self.eps = eps
+    self.affine = affine
+    if self.affine:
+      self.weight = Parameter(Tensor(num_channels))
+      self.bias = Parameter(Tensor(num_channels))
+    else:
+      self.register_parameter('weight', None)
+      self.register_parameter('bias', None)
+    self.reset_parameters()
+
+  def reset_parameters(self) -> None:
+    if self.affine:
+      init.ones_(self.weight)
+      init.zeros_(self.bias)
+
+  def import_params_torch_to_returnn(self, *, layer: NormLayer, torch_module: GroupNorm):
+    assert isinstance(layer, NormLayer)
+    session = tf.compat.v1.get_default_session()
+    layer.params["scale"].load(torch_module.weight.detach().numpy(), session=session)
```

Comment:
There should be a check `if self.affine`, right?

Reply:
Absolutely